core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableConvention.java

```diff
@@ -62,6 +62,6 @@ public boolean canConvertConvention(Convention toConvention) {
 
   public boolean useAbstractConvertersForConversion(RelTraitSet fromTraits,
       RelTraitSet toTraits) {
-    return false;
+    return !fromTraits.containsIfApplicable(Convention.NONE);
   }
```

Comment:
Should this be the default implementation ?

Reply:
It could be. I wish so.